apps/filebrowser/src/filebrowser/views.py

```diff
@@ -125,11 +138,11 @@
 
 INLINE_DISPLAY_MIMETYPE = re.compile(
     'video/|image/|audio/|application/pdf|application/msword|application/excel|'
-    'application/vnd\.ms|'
-    'application/vnd\.openxmlformats'
+    r'application/vnd\.ms|'
+    r'application/vnd\.openxmlformats'
```

Comment:
I feel this one is a false positive and first we should make the whole string in oneline and then let ruff correct it.

This way it feels like line 141 and 142 is separate and line 140 is not starting with `r''`

What do you think?

Reply:
Hmm, yeah makes sense. Made it to a single line.